system/assets/hypha-image-uploader.js

```diff
@@ -0,0 +1,55 @@
+$(function() {
+	$('[data-image-uploader]').each(function () {
+		// get input
+		var $input = $(this);
+		console.log($input.data('image-uploader-src'));
+
+		var src = $input.data('image-uploader-src');
+
+		// Add preview
+		var $preview;
+
+		$input.on('change', function () {
+			createPreviewImage(window.URL.createObjectURL(this.files[0]));
+			toggleInputAndDelete();
+		});
+		var createPreviewImage = function (src) {
+			if (src) {
+				$preview = $('<img class="preview_uploaded_image" src="' + src + '">').insertAfter($input);
```

Comment:
This probably needs some htmlencoding for the src attribute (probably easiest to generate an img with src and then use the jquery attribute set method to set the src).

Reply:
fixed